install/scripts/docker_entrypoint.sh

```diff
@@ -13,13 +13,34 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-# Usage
-# (sudo if needed) docker exec ${CONTAINER_ID} /forseti-security/install/scripts/docker_entrypoint.sh ${BUCKET}
-
 # UNDER DEVELOPMENT, FOR PROOF OF CONCEPT PURPOSES ONLY
 # This script serves as the entrypoint for starting Forseti server or client in a Docker container.
 # Ref. https://docs.docker.com/engine/reference/builder/#entrypoint
 
+# Usage
+# <sudo> docker exec ${CONTAINER_ID} /forseti-security/install/scripts/docker_entrypoint.sh
+# --bucket <bucket>                             the Forseti GCS bucket containing configuration files etc
+# --log_level <info,debug,etc>                  the Forseti server log level
+# --run_server                                  start the Forseti server
+# --services <list of services>                 over-ride default services "scanner model inventory explain notifier"
+# --run_client                                  just provide a container to run client commands
+# --run_cronjob                                 run the cronjob immediately after starting server, for k8s CronJob
+# --sql_host <host ip>                          over-ride k8s (CLOUDSQLPROXY_SERVICE_HOST), cos (localhost) default
+# --sql_port <port>                             over-ride k8s (CLOUDSQLPOXY_SERVICE_PORT), cos (3306) default
+
+# Use cases
```

Comment:
@blueandgold these are the use cases we plan to cover for the experimental version. please let me know if you have any questions.